File: DEMTU.py
str_input = str(input()).strip()
check = False
count = 0
for char in range(0,len(str_input)):
    if str_input[char] != " ":
        if check:
            continue
        else:
            count +=1
            check = True
    else:
        check = False
print(count)

File: DEMTU/DEMTU.cpp
#include<bits/stdc++.h>
using namespace std;

int main() {
    string str_input;
    getline(cin, str_input);
    bool check = false;
    int count = 0;
    for (int char_idx = 0; char_idx < str_input.length(); char_idx++) {
        if (str_input[char_idx] != ' ') {
            if (check) {
                continue;
            } else {
                count++;
                check = true;
            }
        } else {
            check = false;
        }
    }
    cout << count << endl;
    return 0;
}
